development/release-notes.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Release Notes
+---
+
+All changes and improvements to Literal are listed here. For changes in the SDKs, go to the [Python SDK](/python-client/development/changelog) or [TypeScript SDK](/typescript-client/development/changelog).
+
+
+# 0.0.603-beta (April 29, 2024)
+
+This version is compatible with:
+* [Chainlit](https://docs.chainlit.io/get-started/overview) version `1.0.504`.
```

Comment:
I would specify the full range of available versions. For instance we are going to release a new version of chainlit (1.505) today and it is compatible

Reply:
Okay so how would I know which versions are compatible? Can I assume 1.504+ are compatible? And does this also count for the clients?

Reply:
Let's say 1.0.504+ for chainlit, 0.0.503+ for typescript client and 0.0.506+ for python client